### PR TITLE
Feature/filters getall with query parameters

### PR DESCRIPTION
**Pull request by luismiguel0529**
## Summary
- Add method in controller
- Add Query in repository
- Add Test Status OK
## Trello Card
https://trello.com/c/GpOofCGR/32-filtros-en-getall-por-query-parameters

### CODE REVIEW

**Review by joredudiaz — CHANGES_REQUESTED** [review]
Test the new method on the repository please 😄

**Comment by luismiguel0529** [comment]
requested test is added, validate please

File: src/main/java/wolox/training/controllers/BookController.java [hunk 1]
```diff
@@ -177,4 +177,43 @@ public void delete(@PathVariable Long id) {
...
+     * @param pages     parameter to search by pages
+     * @param isbn      parameter to search by isbn
+     * @return book depending on the parameters
+     */
+    @ApiOperation(value = "Method to search a book by all parameters", response = Book.class)
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Book found successfully"),
+            @ApiResponse(code = 404, message = "Book not found")
+    })
+    @GetMapping("/parameters")
```
Comment (facucompe) [inline src/main/java/wolox/training/controllers/BookController.java]: Maybe you should use the index endpoint and add this parameters

**Review by facucompe — CHANGES_REQUESTED** [review]

**Review by joredudiaz — APPROVED** [review]
😄

**Review by facucompe — APPROVED** [review]

File: src/main/java/wolox/training/controllers/BookController.java (hunk 1 above)
Comment (facucompe) [reply src/main/java/wolox/training/controllers/BookController.java]: Se desestima comentario para dejar de manera explicita las distintas maneras de hacer queries en los repositorios

**Pull request merged**